src/main.c

```diff
@@ -12,16 +12,22 @@
 void internal_clock();
 
 void init_all(){
-    //put all gpio and peripheral initilizations in here
+    //put all gpio and peripheral initilizations in here  
```

Comment:
make another commit with empty main function -- it should be ok because you have this commit to log your test cases